### PR TITLE
Update python to 3.9

### PR DESCRIPTION
**Pull request by berroar**
### What is the context of this PR?
Updates the Python version to 3.9

### How to review 
Check that nothing is broken/unit tests still pass

### CODE REVIEW

**Comment by berroar** [comment]
[Python 3.9 (L)](https://trello.com/c/sGLihMaz/4198-python-39-l)

File: eq_translations/translatable_item.py [hunk 1]
```diff
@@ -1,7 +1,7 @@
 from dataclasses import dataclass
 from typing import Union, Optional, List
 
-
+# pylint: disable=unsubscriptable-object
```
Comment (petechd) [inline eq_translations/translatable_item.py:4]: This happens to previous versions of pylint we used(2.5.2) after update to 3.9.

**Review by petechd — APPROVED** [review]

File: eq_translations/translatable_item.py (hunk 1 above)
Comment (MebinAbraham) [inline eq_translations/translatable_item.py:4]: Is this actually needed?. I removed it and the lint passed file with 3.9 with latest lock file.

Comment (berroar) [reply eq_translations/translatable_item.py]: Just reverted and the build is falling over again, so not sure why yours is passing and mine is not 😐

Comment (petechd) [reply eq_translations/translatable_item.py]: Failed in my local 3.9 build too.

Comment (MebinAbraham) [reply eq_translations/translatable_item.py]: Ya my mistake, can reproduce it now. Open issue: https://github.com/PyCQA/pylint/issues/3882

**Review by MebinAbraham — APPROVED** [review]

**Pull request merged**